examples/python/interceptors/streaming/hellostreamingworld_pb2.pyi

```diff
@@ -0,0 +1,67 @@
+"""
```

Comment:
These generated files should be added to [the Makefile](https://github.com/grpc/grpc/blob/master/examples/python/Makefile).